src/components/CardCompanyComponent.tsx

```diff
@@ -3,14 +3,15 @@ import {View, Image, StyleSheet} from 'react-native';
 import {TextComponent} from '../components';
 import {Company} from '../interface/companyinterface';
 import {globalStyles} from '../theme/globalTheme';
+import {TouchableOpacity} from 'react-native-gesture-handler';
```

Comment:
change touchableOpacity, is imported from react- native dont from react-native-gesture-handler